tarantool/response.py

```diff
@@ -248,3 +251,34 @@ def __str__(self):
         return ''.join(output)
 
     __repr__ = __str__
+
+
+class ResponseExecute(Response):
+    @property
+    def lastrowid(self):
+        if self.body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+        info = self.body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return None
+
+        lastrowids = info.get(IPROTO_SQL_INFO_AUTOINCREMENT_IDS)
+
+        return lastrowids[-1] if lastrowids else None
+
+    @property
+    def rowcount(self):
+        if self._body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+
+        info = self._body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return -1
+
+        return info.get(IPROTO_SQL_INFO_ROW_COUNT, -1)
```

Comment:
Yes, I agree that -1 value looks weird in case of Connection -> Tarantool interaction. I changed returned value to None and handle result in .execute() method: `self._rowcount = response.affected_row_count or -1`